cluster/images/etcd/rollback/rollback.go

```diff
@@ -26,7 +26,10 @@ import (
 	"strings"
 	"time"
 
-	wal237 "k8s.io/kubernetes/third_party/forked/etcd237/wal"
+	// Uncomment when you want to rollback to 2.2.1 version.
+	oldwal "k8s.io/kubernetes/third_party/forked/etcd221/wal"
+	// Uncomment when you want to rollback to 2.3.7 version.
+	// oldwal "k8s.io/kubernetes/third_party/forked/etcd237/wal"
```

Comment:
Can we add import based on flag? If you can suggest how to do it, I can change that (to be honest, I don't know how to do it.)